server/app.py

```diff
@@ -1,4 +1,4 @@
-from flask import Flask
+from flask import Flask, request, current_app
```

Comment:
I don't think this is necessary?

Reply:
This was leftover from some testing, will remove